README.md

```diff
@@ -32,73 +32,131 @@ live preview will be added to each PR.
+| `dispatch_type` | ❌ | choice | `"docs_update"` | Action type: `docs_update`, `docs_merge`, or `docs_close` |
+| `pr_number` | ❌ | string | `-` | adds a link to the docs PR to the original PR in metabase/metabase |
+| `update_dirs` | ❌ | string | `""` | comma delimited string of extra dirs to git add during `docs_update` |
 
-- Since we've split up the site into 2 jekyll instances, we cannot rely on
-  htmlproofer to check links from the docs to the marketing site. So
-  `analyze_links.clj` checks any missing links against `metabase.com`.
-  
-- Copies over control directories from marketing repo: `_data`, `_includes`,
-  `_layouts`, `_plugins`, and `_sass`. These are needed so we get a faithful
-  docs build.
+#### Tips and Best Practices
+
+##### Annotations
+- Use descriptive annotations to identify workflows in the build queue and PR
+  titles
+- Keep them concise since they appear in PR titles
+
+##### Branch Naming
+- Source branch must match exactly what's in metabase/metabase
+- Target branch is typically `master`
+- Use `release-x.MM.x` format for backports (e.g., `release-0.50.x`)
+- The source branch must exist in metabase/metabase when running this workflow.
+  If the source branch has been deleted (e.g., after merging the original PR),
+  the workflow will fail.
 
-#### Triggering
+**Solution:**:
+1. **Restore the branch** in metabase/metabase before running the workflow
 
-The [Process Docs Changes](.github/workflows/process_docs_changes.yml) workflow
-is triggered from `metabase/metabase` (aka the main repo). The triggering
-workflow includes the target and source branch name, e.g.: [`my-docs-hotfix` ->
-`master`] or [`v49-new-feature-dox` -> `release-x.49.x`].
 
-Building docs can be [run
-manually](https://github.com/metabase/docs.metabase.github.io/actions/workflows/process_docs_changes.yml)
-as well.
+### Branch Mapping
+The workflow maps branches from the main repository to documentation changes:
+- **Source → Target**: `docs-update-naming-55` → `master` creates a PR with
+  branch `docs-update-naming-55->master` targeting `master` in this repo.
+- **Backports**: `docs-hotfix-security` → `release-0.50.x` creates a PR with
+  branch `docs-hotfix-security->release-0.50.x` targeting `master` in this repo.
 
-#### New Script Docs
+### Dispatch Types
 
-Note, these scripts take an optional `--dry-run` flag that explains what they do
-without actually doing the operation.
+#### `docs_update` (Default)
+Creates or updates a documentation PR with the naming pattern `[annotation]
+{source-branch}->{target-branch}`. The branchname for the PR will always be
+`{source-branch}->{target-branch}`.
+
+**Example**: Source `docs-new-way-to-query` targeting `master` creates/updates
+PR named `[auto-build]docs-new-way-to-query->master`. This PR, in this repo,
+will have branchname `docs-new-way-to-query->master` and target `master` in this
+repo.
```

Comment:
PRs or branches?

```suggestion
**Example**: 

In the Metabase repo, someone opens a PR. For example, a branch called `docs-edit`. The PR targets the `master` branch.

The Metabase PR will trigger this repo to create a new branch in this repo called `docs-edit->master` and a corresponding PR for that branch called `[auto-build]docs-edit->master` in this repo.
```

Does this cover it?

Reply:
```suggestion
**Example**: 
	
	In the Metabase repo, someone opens a PR. For example, from a branch called `docs-edit` targeting `master`.
	
	The Metabase PR will trigger the `process_docs_changes` workflow in this repo to create a new PR in this repo called `docs-edit->master` and a corresponding PR for that branch named `[auto-build] docs-edit->master` in this repo.
```